site/content/docs/user/ingress-nginx.md

```diff
@@ -0,0 +1,56 @@
+# Ingress Nginx
+
+Ingress Nginx in kind works by exposing ports `80(http)` and `443(https)`
```

Comment:
maybe s/works/can be setup/ -- "works" implies this is the only solution or that we bundle a solution.